crates/bindings-csharp/Runtime/Attrs.cs

```diff
@@ -1,73 +1,87 @@
-﻿namespace SpacetimeDB;
-
-[Flags]
-public enum ColumnAttrs : byte
+﻿namespace SpacetimeDB
 {
```

Comment:
Don't think the toplevel ns needs {}; namespace Internal {} is fine to use with namespace SpacetimeDB; afaik

Reply:
It's necessary here because you can't mix file-scoped and regular namespace syntax, C# will complain. Idk why they made it so, but 🤷‍♂️